sapi/cli/tests/bug71624.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Bug #61977 Test that -R properly sets argi and argn
+--SKIPIF--
+<?php
+include "skipif.inc";
+?>
+--FILE--
+<?php
+
+$php = getenv('TEST_PHP_EXECUTABLE');
+
+$filename_txt = dirname(__FILE__)."/bug71624.test.txt";
+
```

Comment:
For portability as well, DIRECTORY_SEPARATOR should be used, as it's passed to the system shell cmd then.

Thanks.